sympy/calculus/tests/test_singularities.py

```diff
@@ -11,11 +11,16 @@
 a = Symbol('a', negative=True)
 b = Symbol('b', positive=True)
 
+
 def test_singularities():
     x = Symbol('x', real=True)
+    y = Symbol('y', real=False)
```

Comment:
Firstly Also `solveset` is to some-extent designed to be independent of assumptions on symbols.
Secondly: defining `real=False` i the user expect in the [last test](https://github.com/sympy/sympy/pull/10275/files#diff-5a88273f772609bb2b362a098bb05c97R22) to not have the output `-1`.
So i think you can remove the assumptions from both `x` and also from `y`, since `solveset` is assumption independent (though not necessarily for inequalities).